listenbrainz/spark/spark_reader.py

```diff
@@ -0,0 +1,100 @@
+
+import json
+import logging
+import pika
+import time
+import ujson
+import os
+
+from flask import current_app
+from listenbrainz import utils
+from listenbrainz.db import user as db_user, stats as db_stats
+from listenbrainz.webserver import create_app
+from listenbrainz.db.exceptions import DatabaseException
+from listenbrainz import config
+import sqlalchemy
+
+TEST_DATA_PATH = os.path.join('/code', 'listenbrainz', 'listenbrainz', 'testdata/')
```

Comment:
what's `testdata` in this case? This doesn't sound like something that should be in a production script. We shouldn't store runtime data in the source directory. This should be in a separate volume, and the configuration for the path location should be in the config file.

Reply:
The YearMonth would be imported from Listenbrainz-playground and the database would look like "2019-01" : {artist_stats} and so on. This value ("2019-01") should be available to test scripts and hence I stored a single JSON file in testdata from which the value can be retrieved. I will make a separate volume though :)

Reply:
I don't understand what you mean by test scripts. Do you mean unit tests? or do you mean experiments that you want to run on the data? Why are you putting it in a file and not in the database?

Reply:
I meant unit tests (./test.sh). I should have been clear with the word. 

Since it is for unit test, I did not saved it in the db.

Reply:
So it sounds like this code shouldn't be here then. If it's just to generate data for using in a test then it shouldn't be run during a production script. Consider making a separate script to just retrieve and download this data if you need.